### ISSUE TITLE
USB-C connector doesn't work with USB-C cables

### ISSUE THREAD

**Issue by viernullvier** [makerbase-mks/MKS-TinyBee#112]
I've been using a TinyBee board as a CNC controller (with FluidNC) for quite some time and in general I am really happy about it.

I've been wondering from the start though why the board would only get recognised when using the supplied USB-C -> USB-A cable, but not with any USB-C -> USB-C cable. Upon further inspection, I could make it work with a couple of adapters: USB-C (computer) -> USB-C-to-A adapter -> USB-A-to-C adapter -> USB-C cable -> USB-C (board).

After a bit more digging, I managed to find the actual reason:

<img width="170" alt="Screenshot 2024-02-11 at 23 24 54" src="https://github.com/makerbase-mks/MKS-TinyBee/assets/1492179/b5c7af52-ea1c-4370-8a61-3123f67a1098">

Both CC pins are not connected, which means that the device can not be recognised as a valid USB-C sink because a connected source can not tell that the device is plugged in at all. The easiest way to make it work would be adding a 5k1 pull-down resistor on each CC pin, which would signal the source to provide 5V at 3A max. Here's an example schematic ([source](https://dubiouscreations.com/2021/04/06/designing-with-usb-c-lessons-learned/#putting-it-all-together-aka-tldr)):

![image-6-450x312](https://github.com/makerbase-mks/MKS-TinyBee/assets/1492179/83d22d5e-d5e2-40ef-8c52-373dc9e0cb1b)

**TL;DR:**
- The current revision of the board will only work with USB-A cables. USB-C cables will not work (without modification).
- In order to make the board work properly with USB-C cables, two 5k1 resistor would need to be added, connecting each CC pin to GND. It would be useful to add them in the next hardware revision.
- Some of the "USB not working" issues (#98, #53, #46) might be caused by this issue.
- Someone braver than me should try bodging the resistors to the CC pins.
